src/lib/formats/sdf_dsk.cpp

```diff
@@ -0,0 +1,202 @@
+// license:BSD-3-Clause
+// copyright-holders:Wilbert Pol, tim lindner
+/*********************************************************************
+
+    formats/sdf_dsk.h
+
+    SDF disk images. Format created by Darren Atkinson for use with
+    his CoCoSDC floppy disk emulator.
+    
+    http://cocosdc.blogspot.com/p/sd-card-socket-sd-card-socket-is-push.html
+
+*********************************************************************/
+
+#include <assert.h>
+
+#include "sdf_dsk.h"
```

Comment:
Please `#include` your module header before other headers to avoid nastiness where headers require you to pre-include other headers.  Yes, I realise you copied from something that does the same thing, but that doesn't make it right ;)